digital-signage-app-v2/app/components/abstract-list.js

```diff
@@ -46,22 +45,21 @@ export default Ember.Component.extend({
   //checks input on key down to see if it is valid
   //This needs to be pulled from the model later, not be hard coded
   keyDown(event) {
```

Comment:
Pretty sure this keyboard logic can be broken into a mixin but not sure. Have to test more.